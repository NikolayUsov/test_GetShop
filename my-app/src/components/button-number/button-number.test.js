import React from 'react';
import { render } from '@testing-library/react';
import ButtonNumber from './button-number';

const handleButtonNumberClick = jest.fn();

describe('Test Button number', () => {
  it('Should create snapshot correctly', () => {
    const { asFragment } = render(<ButtonNumber value="5" handleButtonNumberClick={handleButtonNumberClick} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
