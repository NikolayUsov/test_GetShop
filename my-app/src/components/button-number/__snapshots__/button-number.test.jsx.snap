// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Button number Should create snapshot correctly 1`] = `
<DocumentFragment>
  <button
    class="button"
    type="button"
  />
</DocumentFragment>
`;
